keepassxreboot/keepassxc
# Make preview panel expiration date use local time

## Description

Adds call to toLocalTime() to convert expiration date string
to correct timezone instead of UTC, which matches the date string
in the database view.

## Motivation and context


Fixes #2378

## How has this been tested?



- Create "Expiry Test" entry and set expiration to 10/12/18 12:21 AM
- Verify it appears in local time in the database view
- Verify it appears in local time in the entry preview panel
- Ran existing tests

## Screenshots (if appropriate):
![screenshot from 2018-10-12 00-20-01](https://user-images.githubusercontent.com/5200550/46854219-fe5b3e00-cdb4-11e8-82c4-7d52684a62d9.png)


## Types of changes


- ✅ Bug fix (non-breaking change which fixes an issue)

## Checklist:




- ✅ I have read the **CONTRIBUTING** document. **[REQUIRED]**
- ✅ My code follows the code style of this project. **[REQUIRED]**
- ✅ All new and existing tests passed. **[REQUIRED]**
- ✅ I have compiled and verified my code with `-DWITH_ASAN=ON`. **[REQUIRED]**
